.github/workflows/ci.yaml

```diff
@@ -302,6 +294,7 @@ jobs:
 
       - run: just install_ffigen
 
+      # use action.yaml due to I don't know what was done with the `Install crates`
```

Comment:
uber-nit: Not very sure what this line of comment points to

Reply:
- name: Install crates
  if: steps.test-cache.outputs.cache-hit != 'true'
  run: |
    sh -c "$(curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf)" '' -f

Reply:
Then I guess maybe we should not add this as a comment line?